cocotb/handle.py

```diff
@@ -78,6 +78,8 @@ def __init__(self, handle, path):
 
         self._name = simulator.get_name_string(self._handle)
         self._type = simulator.get_type_string(self._handle)
+        self._is_port = simulator.get_is_port(self._handle)
+        self._port_direction_string = simulator.get_port_direction_string(self._handle)
```

Comment:
Can we make this an enum now that we've dropped Python 2? Could either do that within this file, as
```
from enum import Enum

class PortDirection(Enum):
    UNHANDLED = 0
    INPUT = 1
    OUTPUT = 2
    INOUT = 3
    MIXEDIO = 4  # Verilog only
    NO_DIRECTION = 5  # Verilog only
    BUFFER = 6  # VHDL only
    LINKAGE = 7  # VHDL only
    INTERNAL = 8  # FLI (Mentor) only, meaning "is not a port"
```
```python
port_dir = simulator.get_port_direction_string(self._handle)
self._port_direction = None if port_dir == -1 else PortDirection(port_dir)
```